Refresher/Patching/EbootPatcher.cs

```diff
@@ -96,17 +120,102 @@ private static List<PatchTargetInfo> FindPatchableElements(Stream file)
         List<PatchTargetInfo> foundItems = new();
         FilterValidUrls(reader, possibleUrls, foundItems);
         FindDigestAroundCookie(reader, cookiePositions, foundItems);
+        FindLbpkDomains(reader, lbpkPositions, foundItems);
+        FindLbpkPorts(reader, lbpkPortPositions, foundItems);
```

Comment:
LITTLEBIG000001 does not appear in the LBPK EBOOT

Reply:
Yeah, that was an example from a mainline game. It'll be a different string depending on the game.

Reply:
I could just base it off "did we find an lbpk domain", thats probably the most bulletproof way to determine if this is LBPK, since the patch only makes sense in that game specifically

Reply:
Hmm, but then you wouldn't be able to repatch. Does LBPK have anything starting with EP9000, HP9000, or UP9000? Serialstation has a couple content ids listed for karting: https://serialstation.com/contents/ids?name=littlebigplanet%20karting&types=e72dffb4-f545-4dc1-8a8c-49988262a45c

Reply:
We already are unable to repatch LBPK for a couple reasons, 
a) the offsets of domains and ports change depending on the version (digital -> disk), so we cant easily hardcode offsets
b) domains are *extremely* vague compared to a well-formed HTTP URL, so we'd have to require the user to input the previous domain so we can search for it
c) theres no heuristic i can think of to know where the ports are in the EBOOT (i mean i can find the specific code that references the ports using byte checking, and just follow where that points, but i feel that would be even less reliable than the current method) aside from checking for the 2 numbers, so we'd need to make users input the previous 2 ports to be able to repatch

Reply:
I see, fair enough then. Yeah, go ahead and just search for `lbpk` then. :+1:

Reply:
assuming this is resolved now